ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -7,6 +7,10 @@ base_images:
     name: ocm-2.13
     namespace: edge-infrastructure
     tag: assisted-service-scripts
+  aws-efs-csi-operator-create-efs:
```

Comment:
It seems this image also needed in 4.19 upgrade jobs, right?
```
openshift-openshift-tests-private-release-4.19__arm64-nightly-4.19-upgrade-from-stable-4.18.yaml:- as: aws-ipi-disc-priv-sts-efs-f28
openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.18.yaml:- as: aws-ipi-disc-priv-sts-efs-f28
```

Reply:
Yeah, correct. Good catch, thanks! I did not check the 4.19 jobs. Let me also update the 4.19 config.

Reply:
Fixed.